### PR TITLE
Fixing erroneous community PR

### PR DESCRIPTION
**Pull request by alexellis**
This was submitted originally some time ago, I saw it had an erroneous update so wanted to make a quick correction. Thanks everyone for this cool list 👍 

**[Insert the URL to be listed]**

**[Explain what the project is about and why it should be listed]**

# By submitting this pull request I confirm I've read and complied with the below requirements.

**Please read it multiple times. I spent a lot of time on these guidelines and most people miss a lot.**

- I have read and understood the [contribution guidelines](https://github.com/veggiemonk/awesome-docker/blob/master/CONTRIBUTING.md)

- The project submitted is conform to the quality standards outlined in the [contribution guidelines](https://github.com/veggiemonk/awesome-docker/blob/master/CONTRIBUTING.md)

- Go to the [contribution guidelines](https://github.com/veggiemonk/awesome-docker/blob/master/CONTRIBUTING.md) and read it again.

**RULES (because you probably didn't read the guidelines):**

Links will be tested with [awesome_bot](https://github.com/dkhamsing/awesome_bot). Please make sure your URL to be listed is unique (do some search queries), otherwise put a link at the end of `README.md` file.
 
- FORMAT
  - Use the following format: `[Name](Link) - Description.`
  - `[Name]` should be the title of the project
  - `(Link)` should be pointing to a GitHub repository, not to a website
  - `Description.` should end with a full stop/period
  - For GitHub repositories, add the name of the author with a link to its GitHub account after the description, like `By [@author](GitHub account)`
- ORDER: The entries are in lexical order (English letter order) 
- LABEL (Optional): Add one of the following icons next to the project name like ```[Name](Link) :icon: - Description.```
  - :heavy_dollar_sign: indicates if one needs a paying subscription to use the project.
  - :construction: indicates that the project is a work in progress/beta release.
  - :skull: indicates that the project is obsolete, but still worth mentioning it.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -446,7 +446,7 @@ Services to securely store your Docker images.
 * [AMP](https://github.com/appcelerator/amp) - The open source unified CaaS/FaaS platform for Docker, batteries included. By [@Appcelerator](https://github.com/appcelerator/)
 * [Apache OpenWhisk](https://github.com/apache/incubator-openwhisk) - a serverless, open source cloud platform that executes functions in response to events at any scale. By [@apache](https://github.com/apache)
 * [Docker-Lambda](https://github.com/lambci/docker-lambda) - Docker images and test runners that replicate the live AWS Lambda environment. By [@lamb-ci](https://github.com/lambci)
-* [FaaS](https://github.com/openfaas/faas) - Docker Serverless/Functions as a Service. By [@openfaas](https://github.com/openfaas/)
+* [OpenFaaS](https://github.com/openfaas/faas) - A complete serverless functions framework for Docker and Kubernetes. Twitter: [@open_faas](https://github.com/open_faas/), Web: [openfaas.com](https://www.openfaas.com)
```
Comment (devhkr) [inline README.md]: @alexellis thanks for your contribution. Actually, we prefer GitHub link only in this case. Please also list the tool in alphabetical order.

**Comment by alexellis** [comment]
I'm not at the pc. I'm happy for you to edit.

**Comment by alexellis** [comment]
It needs fixing though

**Comment by devhkr** [comment]
@agebhar1 thanks for the hint. Yes, it's obvious here. I gotta confused because the repo name is github.com/openfaas/**faas** but in the official description it's OpenFaaS instead.

**Comment by devhkr** [comment]
@alexellis sorry for the inconvenience. I kinda have misunderstood your statement
>[...]. I'm happy for you to edit.

I thought, you gave me the permission to edit.

File: README.md [hunk 2]
```diff
@@ -446,9 +446,9 @@ Services to securely store your Docker images.
...
 * [Apache OpenWhisk](https://github.com/apache/incubator-openwhisk) - a serverless, open source cloud platform that executes functions in response to events at any scale. By [@apache](https://github.com/apache)
 * [Docker-Lambda](https://github.com/lambci/docker-lambda) - Docker images and test runners that replicate the live AWS Lambda environment. By [@lamb-ci](https://github.com/lambci)
-* [FaaS](https://github.com/openfaas/faas) - Docker Serverless/Functions as a Service. By [@openfaas](https://github.com/openfaas/)
 * [Funker](https://github.com/bfirsh/funker-example-voting-app) - Functions as Docker containers example voting app. By [@bfirsh](https://github.com/bfirsh)
 * [IronFunctions](https://github.com/iron-io/functions) - The serverless microservices platform FaaS (Funcitons as a Service) which uses Docker containers to run Any language or AWS Lambda functions
+* [OpenFaaS](https://github.com/openfaas/faas) - A complete serverless functions framework for Docker and Kubernetes. By [OpenFaaS](https://github.com/openfaas)
```
Comment (veggiemonk) [inline README.md]: This sounds good to me.

**Pull request merged**